python/lsst/daf/butler/core/formatter.py

```diff
@@ -289,7 +288,7 @@ def getLookupKeys(self):
         """
         return self._mappingFactory.getLookupKeys()
 
-    def getFormatterClassWithMatch(self, entity):
+    def getFormatterClassWithMatch(self, entity) -> Tuple[LookupKey, Type]:
```

Comment:
A `typing.Union`-based alias for the `entity` arguments would be pretty nice, I think.